### DISCUSSION TITLE
zotero-style在输入easyscholar密钥后仍无法显示期刊标签

### DISCUSSION THREAD

**Discussion by 13145699000** [MuiseDestiny/zotero-style#548] [General]
输入easyscholar密钥后仍无法显示期刊标签，点击upstate publication tags后显示❌【0/0/5/5】100％。请求解答，谢谢！

**Comment by MuiseDestiny** [comment]
你的密钥输入到哪里了。截图，打码

**Reply by 13145699000** [reply]
![微信图片_20230809171146](https://github.com/MuiseDestiny/ethereal-style/assets/141660614/2eaab609-8a7f-4441-9264-f991b1e945a0)
![微信图片_20230809172527](https://github.com/MuiseDestiny/ethereal-style/assets/141660614/d2d3004a-da05-49ae-93da-d86ec4fa246c)

**Reply by 13145699000** [reply]
抱歉昨天回复太晚了，能麻烦您抽空再回复一下吗 @MuiseDestiny ，万分感谢！

**Comment by 13145699000** [comment]
![微信图片_20230809171146](https://github.com/MuiseDestiny/ethereal-style/assets/141660614/6ecea87a-7402-4b44-a367-dbd73d78da4f)

**Comment by 13145699000** [comment]
![微信图片_20230809172527](https://github.com/MuiseDestiny/ethereal-style/assets/141660614/4cf54f82-7c9d-43ca-8915-13d4f2381f66)

**Comment by somili1017** [comment]
您好！我也是这种情况 请问你解决了吗
